unipop-core/src/org/unipop/controller/EdgeController.java

```diff
@@ -8,7 +8,14 @@
 import java.util.*;
 
 public interface EdgeController {
-    Iterator<BaseEdge> edges(Predicates predicates, MutableMetrics metrics);
```

Comment:
It's a tree of metrics used by the Profile step. So when someone doing
g.V().profile() can get some more details from the different controllers.
It hasn't been used anywhere yet, but it should be useful.

On Sat, 14 Nov 2015 at 22:03 giladovich <notifications@github.com> wrote:

> In unipop-core/src/org/unipop/controller/EdgeController.java
> <https://github.com/rmagen/unipop/pull/5#discussion_r44862862>:
>
> > @@ -8,7 +8,14 @@
> >  import java.util.*;
> >
> >  public interface EdgeController {
> > -    Iterator<BaseEdge> edges(Predicates predicates, MutableMetrics metrics);
>
> What was the purpose of it? We couldn't find any usages nor reason for
> putting it there.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rmagen/unipop/pull/5/files#r44862862>.
>